.github/workflows/go.yml

```diff
@@ -32,3 +31,14 @@ jobs:
       with:
         name: test-coverage
         path: coverage.html
+
+
+  lint-code:
+    name: Lint code
+    runs-on: ubuntu-latest
```

Comment:
I don't know how to add a github action. So my questions are for learning. What is the reason for specifying ubuntu-latest?

Reply:
it's the flavour that GH uses.

Reply:
great, thanks.